Assets/Mirage/Runtime/NetworkWorld.cs

```diff
@@ -25,6 +25,7 @@ public class NetworkWorld : IObjectLocator
         public NetworkTime Time { get; } = new NetworkTime();
 
         private readonly Dictionary<uint, NetworkIdentity> SpawnedObjects = new Dictionary<uint, NetworkIdentity>();
+        private IReadOnlyCollection<NetworkIdentity> removalCollection;
```

Comment:
this can be a local variable instead of a field

Reply:
This one of my down falls so lets discuss so i know. Would a local variable versus just pooling this way cause more or less gc? I am pretty sure both do so but which one is less performance? 
Other then performance here not sure why it would matter if local or not local?